### PR TITLE
*: refined contributing doc

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added a checklist and style guide.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,46 +2,155 @@
 
 无论你是热爱技术的程序员，还是擅长书面表达的语言爱好者，亦或是纯粹想帮 TiDB 改进文档的热心小伙伴，都欢迎来为 TiDB 文档做贡献，一起打造更加易用友好的 TiDB 文档！
```
Comment (hfxsd) [inline CONTRIBUTING.md:3]: 我们有 code of conduct， terms of use 之类的内容吗，也可以加上链接。参考 https://learn.microsoft.com/en-us/contribute/how-to-write-workflows-major

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,46 +2,155 @@
...
+> - 绝大多数情况下，中英版需要保持完全一致。但个别文档由于受众不同，可能会有差异。
+> - 通常，TiDB 文档先完成中文版后再完成英文版。但也偶有例外。
+> - [参考资料](#参考资料)一节中汇总了**中英术语表**和**风格指南**等参考文档，建议译前阅读。
 
-#### 中文翻译任务的认领方式
+关于如何认领翻译任务的详细步骤，请参见[参考资料](#参考资料)。
+
+## 如何提 Pull Request？
+
+最常见的贡献方式就是提 Pull Request 了，那么提交流程是怎样的，又需要遵守哪些规范呢？请参考下面视频教程，快速上手 GitHub 的 Pull Request 流程。
```
Comment (lilin90) [inline CONTRIBUTING.md:41]:
```suggestion
最常见的贡献方式就是提 Pull Request 了，那么提交流程是怎样的，又需要遵守哪些规范呢？下面的视频教程可以帮你快速上手 GitHub 的 Pull Request 流程。
```

**Review by lilin90 — COMMENTED** [review]
I have not finished the review yet and will continue soon.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,107 +2,110 @@
...
+**PR 的后续跟进（5 分钟）**
 
-3. 修改 PR 标签
+<video src="https://download.pingcap.com/docs-cn/GTV_2_zh_PR_Follow_up.mp4" width="600px" height="450px" controls="controls" poster="https://download.pingcap.com/docs-cn/pr_01_follow_up.png"></video>
 
-    PR 认领成功后，继续在底部 comment 区域依次发送：`/remove-translation welcome` 及 `/translation doing`，即可将右侧 label 栏中的 `translation/welcome` 改为 `translation/doing`，之后你便可以开始翻译了。
+**批量修改建议和 CI 检查（4 分钟）**
 
-4. 翻译 PR 并提交
+<video src="https://download.pingcap.com/docs-cn/GTV_3_zh_Batch_Suggestion_and_CI_Check.mp4" width="600px" height="450px" controls="controls" poster="https://download.pingcap.com/docs-cn/pr_01_suggestions.png"></video>
```
Comment (lilin90) [inline CONTRIBUTING.md:57]: These videos cannot display well. Please check via "**View file**" and update them.
image

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -211,6 +229,50 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
...
+- [PingCAP 中英术语表](https://shimo.im/sheets/tTRyydP8Xkdv8yxq/MODOC)
+- [GitHub Docs](https://docs.github.com/en)
+- [Pull Request Commit Message 规范](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md#how-to-write-a-good-commit-message)
+- [Pull Request 标题规范](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md#pull-request-title-style)
+- [代码注释规范](https://github.com/pingcap/community/blob/master/contributors/code-comment-style.md)
+- [Figma 快速上手教程](/resources/figma-quick-start-guide.md)
+
 ## 联系我们
 
 欢迎来 [TiDB Internals](https://internals.tidb.io/) 参与讨论。
```
Comment (lilin90) [inline CONTRIBUTING.md:278]: TiDB Internals is the hub for discussing everything related to the implementation and design of the TiDB projects, not so appropriate for docs discussion.
```suggestion
```

**Comment by lilin90** [comment]
@hfxsd Please take a look at my commits and comments. Thanks!

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -211,6 +229,50 @@ TiDB 中文文档使用 Markdown 语言进行编写，为了保证文档质量
...
+
+其他快速上手资源
+
+- [PingCAP 中英术语表](https://shimo.im/sheets/tTRyydP8Xkdv8yxq/MODOC)
+- [GitHub Docs](https://docs.github.com/en)
+- [Pull Request Commit Message 规范](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md#how-to-write-a-good-commit-message)
+- [Pull Request 标题规范](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md#pull-request-title-style)
+- [代码注释规范](https://github.com/pingcap/community/blob/master/contributors/code-comment-style.md)
+- [Figma 快速上手教程](/resources/figma-quick-start-guide.md)
+
```
Comment (ran-huang) [inline CONTRIBUTING.md:275]:
```suggestion
- [EBNF 语法图在线渲染](https://kennytm.github.io/website-docs/dist/).

```

File: CONTRIBUTING.md (hunk 3 above)
Comment (hfxsd) [reply CONTRIBUTING.md:57]: 视频没法预览的

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -2,107 +2,110 @@
...
-    - [常用 bot 命令](#常用-bot-命令)
 - [PingCAP 中文文档风格指南](/resources/pingcap-style-guide-zh.pdf)
-- [PingCAP 中英术语表](https://shimo.im/sheets/tTRyydP8Xkdv8yxq/MODOC)
 - [TiDB 中文用户文档模板](/resources/doc-templates)
-- [必须遵循的 Markdown 规范](#必须遵循的-markdown-规范)
-- [代码注释规范](https://github.com/pingcap/community/blob/master/contributors/code-comment-style.md)
-- 图片风格：[Figma 快速上手教程](/resources/figma-quick-start-guide.md)
 
-    为确保文档图片风格统一，建议使用 Figma 绘制图片。绘制图片时，请参考模板提供的图形元素和配色方案。
+## 参考资料
```
Comment (hfxsd) [inline CONTRIBUTING.md:88]: 添加 PR 必须有的 label

File: CONTRIBUTING.md (hunk 3 above)
Comment (lilin90) [reply CONTRIBUTING.md:57]: > 视频没法预览的

@hfxsd 至少需要保证能正常点击观看，或者用最简单的 Markdown 链接语法。

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -2,107 +2,110 @@
...
-你可以通过以下简单几步来认领并提交一个 PR 翻译任务：
+最常见的贡献方式就是提 Pull Request 了，那么提交流程是怎样的，又需要遵守哪些规范呢？下面的视频教程可以帮你快速上手 GitHub 的 Pull Request 流程。
 
-> **注意：**
->
-> 关于下面步骤中所提到的 comment 式命令，详细说明请参考[常用 bot 命令](#常用-bot-命令)。
+**基本概念篇（4 分钟）**
 
-1. 查看待认领 PR
+<video src="https://download.pingcap.com/docs-cn%2FLesson10_config.mp4" width="600px" height="450px" controls="controls" poster="https://download.pingcap.com/docs-cn/pr_00_basic_concepts.png"></video>
```
Comment (hfxsd) [inline CONTRIBUTING.md:45]:
```suggestion
<video src="https://download.pingcap.com/docs-cn/GTV_0_zh_Basic_concepts.mp4" width="600px" height="450px" controls="controls" poster="https://download.pingcap.com/docs-cn/pr_00_basic_concepts.png"></video>
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -2,107 +2,110 @@
...
 
 你可从以下任一方面入手：
 
-- 修复文档格式（如标点、空格、缩进、代码块等）和错别字
-- 修改过时或不当的内容描述
-- 增加缺失的文档内容
+- 更新过时内容
+- 补充缺失内容
+- 修正文档格式，如标点、空格、缩进、代码块等
+- 改正错别字
```
Comment (qiancai) [inline CONTRIBUTING.md:23]:
```suggestion
- 提 [Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) (PR) 更新过时内容
- 提 PR 补充缺失内容
- 提 PR 修正文档格式，如标点、空格、缩进、代码块等
- 提 PR 改正错别字
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -2,107 +2,110 @@
...
 - 回复或解决 [issue](https://github.com/pingcap/docs-cn/issues?q=is%3Aopen+is%3Aissue) 并提 PR 更新相关文档
 - 其它改进
 
 ### 翻译中文文档
 
-TiDB 中文文档的日常更新特别活跃，相应地，[TiDB 英文文档](https://docs.pingcap.com/tidb/dev/)也需要频繁更新。这一过程会涉及很多的**中译英**，即将 pingcap/docs-cn 仓库里已 merge 但尚未进行翻译处理的 Pull Request 翻译为英文，并提交 Pull Request 至 [pingcap/docs 仓库](https://github.com/pingcap/docs)中。**具体的认领方式**如下。
+TiDB 中文文档的日常更新特别活跃，相应地，[TiDB 英文文档](https://docs.pingcap.com/tidb/dev/)也需要频繁更新。这一过程会涉及很多的**中译英**，即将 pingcap/docs-cn 仓库里已 merge 但尚未进行翻译处理的 [Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) (PR) 翻译为英文，并在 [pingcap/docs 仓库](https://github.com/pingcap/docs)中提交 Pull Request。
```
Comment (qiancai) [inline CONTRIBUTING.md:29]:
```suggestion
TiDB 中文文档的日常更新特别活跃，相应地，[TiDB 英文文档](https://docs.pingcap.com/tidb/dev/)也需要频繁更新。这一过程会涉及很多的**中译英**，即将 pingcap/docs-cn 仓库里已 merge 但尚未进行翻译处理的 [PR](https://github.com/pingcap/docs-cn/pulls?q=is%3Apr+label%3Atranslation%2Fwelcome+is%3Aclosed) 翻译为英文，并在 [pingcap/docs 仓库](https://github.com/pingcap/docs)中提交 Pull Request。
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -2,107 +2,110 @@
...
 
 > **注意：**
 >
-> - 由于受众不同，TiDB 的中文文档与英文文档并非完全相同。但绝大多数情况下，中英版本会保持一致。
-> - 通常，TiDB 文档是先有中文版，后有英文版。但也有一小部分文档，是先有英文版，后有中文版。
-> - [快速上手资源](#快速上手资源)一节中汇总了**中英术语表**和**风格指南**等参考文档，建议译前阅读。
+> - 绝大多数情况下，中英版需要保持完全一致。但个别文档由于受众不同，可能会有差异。
```
Comment (qiancai) [inline CONTRIBUTING.md:33]:
```suggestion
> - 绝大多数情况下，中英文档需要保持完全一致。但个别文档由于受众不同，可能会有差异。
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -2,107 +2,110 @@
...
+- [ ] 文档内容准确、清晰、简洁，遵循写作规范。参考 [PingCAP 中文技术文档风格 — 极简指南](#pingcap-中文技术文档风格--极简指南)。
+- [ ] PR 的各元素完整、准确，包括：
+    - [ ] 标题清晰、有意义，包括修改的类型+文档所属的模块。参考 [Commit Message and Pull Request Style](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md)。例如：
+        - Fix typos in tidb-monitoring-api.md
+        - benchmark: add the v5.3.0 benchmark document
+    - [ ] 有简要描述，例如修改背景等，并添加对应的 issue 号（如果有）
+    - [ ] 选择正确的标签 (label)
+    - [ ] cherry-pick 到适用且必要的分支版本
+- [ ] 如果新增文档、删除文档，需要同时更新 `TOC.md`，删除文档时需要在文件开头添加 `aliases` 确保旧链接能够正常跳转。
+- [ ] PR 经过了至少两轮 review，包括 technical review 和 editorial review，确保技术准确、语言规范。
```
Comment (qiancai) [inline CONTRIBUTING.md:74]: L74、L75、L77 在提 PR 之前无法确保

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -2,107 +2,110 @@
...
-## Pull Request 提交流程
+<details>
+<summary>提交 Pull Request 的详细流程</summary>
 
 TiDB 文档的修改需要遵循一定的流程，具体如下。考虑到有些小伙伴是纯语言背景，命令行的流程掌握起来可能需要花些时间，之后我们也会提供更适合小白上手的 GitHub Desktop 客户端版提交流程（在添加至这里之前，可暂时参考 [lilin90](https://github.com/lilin90) 撰写的[小白上手流程](https://zhuanlan.zhihu.com/p/64880410)）。
 
 > **注意：**
 >
-> 目前 TiDB 主要维护以下几个版本的文档：dev（最新开发版，对应文档仓库的 master 分支）、v6.1、v6.0、v5.4、v5.3、v5.2、v5.1、v5.0、v4.0、v3.1、v3.0 以及 v2.1。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。选择版本时，请参考[版本选择指南](#版本选择指南)。
+> 目前（2023 年 3 月）TiDB 主要维护以下几个版本的文档：dev（最新开发版，对应文档仓库的 master 分支）、v6.6、v6.5、v6.4、v6.1、v6.0、v5.4、v5.3、v5.2、v5.1、v5.0。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。选择版本时，请参考[参考资料](#参考资料)中的**如何选择文档适用的版本分支？**。
```
Comment (qiancai) [inline CONTRIBUTING.md:97]:
```suggestion
> 目前（2023 年 12 月）TiDB 主要维护以下几个版本的文档：dev（最新开发版，对应文档仓库的 master 分支）、v7.5、v7.4、v7.3、v7.1、v6.5、v6.1、v6.0、v5.4、v5.3、v5.2、v5.1、v5.0。提 Pull Request 前请务必考虑修改会影响的文档版本，并据此修改所有相应的版本。选择版本时，请参考[参考资料](#参考资料)中的**如何选择文档适用的版本分支？**。
```

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -144,65 +147,80 @@ git commit -m "commit-message: update the xx"
...
+**第 8 步：创建一个 Pull Request**
 
-1. 打开你 Fork 的仓库：<https://github.com/$user/docs-cn>（将 `$user` 替换为你的 GitHub ID）
+1. 打开你 Fork 的仓库：`https://github.com/$user/docs-cn`（将其中的 `$user` 替换为你的 GitHub ID）
 2. 点击 `Compare & pull request` 按钮即可创建 PR。参考[如何写 PR title 和描述](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md)。
 
 > **注意：**
 >
-> - 如果你的修改影响多个文档版本 (dev, v5.0, v4.0, v3.1, v3.0, v2.1)，务必**在 PR 描述框中勾选相应的版本**，后续仓库管理员会为你的 PR 打上相应的 cherry-pick 标签。
+> - 如果你的修改影响多个文档版本 (如 dev、v6.6、v6.5等)，务必**在 PR 描述框中勾选相应的版本**，后续仓库管理员会为你的 PR 打上相应的 cherry-pick 标签。
```
Comment (qiancai) [inline CONTRIBUTING.md:171]:
```suggestion
> - 如果你的修改影响多个文档版本 (如 dev、v7.5、v7.4 等)，务必**在 PR 描述框中勾选相应的版本**，后续仓库管理员会为你的 PR 打上相应的 cherry-pick 标签。
```

File: CONTRIBUTING.md (hunk 11 above)
Comment (hfxsd) [inline CONTRIBUTING.md:74]:
```suggestion
```

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -2,107 +2,110 @@
...
+<video src="https://download.pingcap.com/docs-cn/GTV_3_zh_Batch_Suggestion_and_CI_Check.mp4" width="600px" height="450px" controls="controls" poster="https://download.pingcap.com/docs-cn/pr_01_suggestions.png"></video>
 
-    由于 TiDB 的中英文文档分别存放于 [pingcap/docs-cn](https://github.com/pingcap/docs-cn) 和 [pingcap/docs](https://github.com/pingcap/docs) 中，并且两个仓库的文件结构完全对应。如果你是首次认领翻译任务，需先 fork docs 仓库，并将 fork 的 docs 仓库克隆到本地，然后找到源 PR 中对应的改动文件再开始翻译。翻译完毕后，创建新 PR，将翻译好的文件提交至 docs 仓库。具体操作步骤及更多参考资料可参见下文[快速上手资源](#快速上手资源)一节。
+你也可以查阅 [docs-cn 现有的 Pull Request](https://github.com/pingcap/docs-cn/pulls) 作为参考。关于提 Pull Request 的详细步骤，请查阅[参考资料](#参考资料)。
 
-5. 填写 PR 描述并修改标签
+## PR Checklist
 
-    新建 PR 成功后，先按照模板说明完整填写 PR 描述，接着在底部发送：`/translation from-docs-cn`，为 PR 添加 `translation/from-docs-cn` 标签，表明此 PR 是从中文翻译过来的。然后回到源 PR 依次发送：`/remove-translation doing` 及 `/translation done`，将源 PR 标签修改为 `translation/done`，表明翻译已完成。
+在提交 PR 之前，请务必检查以下内容：
```
Comment (hfxsd) [inline CONTRIBUTING.md:63]:
```suggestion
在合入 (Merge) PR 之前，请务必检查以下内容：
```

File: CONTRIBUTING.md (hunk 11 above)
Comment (hfxsd) [inline CONTRIBUTING.md:74]:
```suggestion
```

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -2,107 +2,110 @@
...
+- [ ] PR 的各元素完整、准确，包括：
+    - [ ] 标题清晰、有意义，包括修改的类型+文档所属的模块。参考 [Commit Message and Pull Request Style](https://github.com/pingcap/community/blob/master/contributors/commit-message-pr-style.md)。例如：
+        - Fix typos in tidb-monitoring-api.md
+        - benchmark: add the v5.3.0 benchmark document
+    - [ ] 有简要描述，例如修改背景等，并添加对应的 issue 号（如果有）
+    - [ ] 选择正确的标签 (label)
+    - [ ] cherry-pick 到适用且必要的分支版本
+- [ ] 如果新增文档、删除文档，需要同时更新 `TOC.md`，删除文档时需要在文件开头添加 `aliases` 确保旧链接能够正常跳转。
+- [ ] PR 经过了至少两轮 review，包括 technical review 和 editorial review，确保技术准确、语言规范。
+- [ ] Comments 全部处理完，且 reviewers 已经 approved 该 PR。
```
Comment (hfxsd) [inline CONTRIBUTING.md:75]:
```suggestion
```

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -2,107 +2,110 @@
...
+- [ ] 预览文档，确保文档格式正确、清晰、可读，特别注意表格、图片、列表等特殊样式能够正常显示。
+- [ ] 文档通过了 CI check。
```
Comment (hfxsd) [inline CONTRIBUTING.md:77]:
```suggestion
```

File: CONTRIBUTING.md (hunk 11 above)
Comment (hfxsd) [reply CONTRIBUTING.md]: deleted

**Review by qiancai — APPROVED** [review]

File: TOC.md [hunk 17]
```diff
@@ -9,6 +9,7 @@
   - [与 MySQL 的兼容性](/mysql-compatibility.md)
   - [使用限制](/tidb-limitations.md)
   - [荣誉列表](/credits.md)
+  - [贡献指南](/CONTRIBUTING.md)
```
Comment (hfxsd) [inline TOC.md:12]:
```suggestion
```

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -2,107 +2,96 @@
...
-
-    打开 [pingcap/docs-cn PR 翻译任务页面](https://github.com/pingcap/docs-cn/pulls?q=is%3Apr+label%3Atranslation%2Fwelcome+)，即可看到所有打上了 `translation/welcome` label 的 PR。这类 PR 无论是处于 open 还是 closed 状态，均可认领。
+## 如何提 Pull Request？
 
-2. 认领 PR
-
-    打开你想认领的 PR，拉到底部留下这条 comment：`/assign`，即可将此 PR 的翻译任务分配给自己。
-
-3. 修改 PR 标签
+最常见的贡献方式就是提 Pull Request 了，那么提交流程是怎样的，又需要遵守哪些规范呢？下面的视频教程可以帮你快速上手 GitHub 的 Pull Request 流程。
```
Comment (lilin90) [inline CONTRIBUTING.md:41]:
```suggestion
最常见的贡献方式就是提 Pull Request 了，那么提交流程是怎样的，又需要遵守哪些规范呢？下面的视频教程可以帮你快速上手 GitHub 的 Pull Request 流程：
```

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -2,107 +2,96 @@
...
 
-    PR 认领成功后，继续在底部 comment 区域依次发送：`/remove-translation welcome` 及 `/translation doing`，即可将右侧 label 栏中的 `translation/welcome` 改为 `translation/doing`，之后你便可以开始翻译了。
+- [Git 与 GitHub 基础概念](https://www.bilibili.com/video/BV1h5411E7pM/?p=1)
+- [如何创建一个 Pull Request (PR)](https://www.bilibili.com/video/BV1h5411E7pM?p=2)
+- [跟进 PR 的后续操作](https://www.bilibili.com/video/BV1h5411E7pM?p=3)
+- [批量接受 Review 建议和处理 CI 检查](https://www.bilibili.com/video/BV1h5411E7pM?p=4)
 
-4. 翻译 PR 并提交
+你也可以查阅 [docs-cn 现有的 Pull Request](https://github.com/pingcap/docs-cn/pulls) 作为参考。关于提 Pull Request 的详细步骤，请查阅[参考资料](#参考资料)。
```
Comment (lilin90) [inline CONTRIBUTING.md:48]:
```suggestion
你也可以查阅 [docs-cn 仓库现有的 Pull Requests](https://github.com/pingcap/docs-cn/pulls) 作为参考。关于提 Pull Request 的详细步骤，请查阅[参考资料](#参考资料)。
```

File: CONTRIBUTING.md [hunk 20]
```diff
@@ -144,65 +133,80 @@ git commit -m "commit-message: update the xx"
...
+
+5. 填写 PR 描述并修改标签
+
+    新建 PR 成功后，先按照模板说明完整填写 PR 描述，接着在底部发送：`/translation from-docs-cn`，为 PR 添加 `translation/from-docs-cn` 标签，表明此 PR 是从中文翻译过来的。然后回到源 PR 依次发送：`/remove-translation doing` 及 `/translation done`，将源 PR 标签修改为 `translation/done`，表明翻译已完成。
+
+6. 分配 Reviewer（推荐，非必需）
+
+    每个 PR 都需要经过 Review 后才能合并，分配 Reviewer 一般由文档仓库管理员负责，但我们也十分欢迎你来主动承担这个任务。
+
+    具体操作为：在新建的 PR 下发送 `/cc @TomShawn @technical-reviewer`（将 technical-reviewer 替换为源 PR 作者的 GitHub ID），即可将 Review 任务分配给 docs 仓库管理员 @TomShawn 及源 PR 的作者。
```
Comment (lilin90) [inline CONTRIBUTING.md:196]:
```suggestion
    具体操作为：在新建的 PR 下发送 `/cc @lilin90 @technical-reviewer`（将 technical-reviewer 替换为源 PR 作者的 GitHub ID），即可将 Review 任务分配给 docs 仓库管理员 @lilin90 及源 PR 的作者。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**